command/polybftmanifest/manifest_init.go

```diff
@@ -168,11 +168,11 @@ func (p *manifestInitParams) getValidatorAccounts() ([]*polybft.Validator, error
 				return nil, fmt.Errorf("invalid address: %s", parts[1])
 			}
 
-			if len(parts[2]) < blsKeyLength {
```

Comment:
Can we handle `0x` prefixes also? Just remove them from the `parts` (by using `strings.TrimPrefix`, I think it is easiest) but you can do that differently

Reply:
I don't think we should do it, if the formatting is consistent in our tooling, that's not necessary, because then, why not do the same in key or other parts?

Reply:
Done in https://github.com/0xPolygon/polygon-edge/pull/1248/commits/72bb83a02cecc0580e54098919919967609b9e6b